recipes/geant4-data-channeling/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set dataset_name = "CHANNELING" %}
+{% set version = "1.0.0" %}
+{% set version_canonical_geant4 = "1.0" %}
+{% set sha256 = "203e3c69984ca09acd181a1d31a9b0efafad4bc12e6c608f0b05e695120d67f2" %}
+{% set env_var = "G4CHANNELINGDATA" %}
+{% set summary = "GEANT4 data files for the simulations of channeling trajectories" %}
+
+{% set name = "geant4-data-"+dataset_name.lower() %}
+{% set install_dir = "share/Geant4/data/"+dataset_name+version_canonical_geant4 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://geant4-data.web.cern.ch/geant4-data/datasets/G4{{ dataset_name }}.{{ version_canonical_geant4 }}.tar.gz
+  sha256: {{ sha256 }}
```

Less Jinja indirection
```suggestion
  sha256: 203e3c69984ca09acd181a1d31a9b0efafad4bc12e6c608f0b05e695120d67f2
```